pkg/triggers/watcher.go

```diff
@@ -0,0 +1,186 @@
+package triggers
+
+import (
+	"context"
+
+	"github.com/google/go-cmp/cmp"
+	testtriggers_v1 "github.com/kubeshop/testkube-operator/apis/testtriggers/v1"
+	"github.com/kubeshop/testkube-operator/pkg/informers/externalversions"
+	"github.com/kubeshop/testkube-operator/pkg/validation/tests/v1/testtrigger"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/tools/cache"
+)
+
+func (s *Service) runWatcher(ctx context.Context) {
+	f := informers.NewSharedInformerFactory(s.clientset, 0)
+	podInformer := f.Core().V1().Pods()
```

Comment:
We could pass them as some array with enabled/disabled property.

Reply:
and then configure which one is handled

Reply:
I enabled all of them, it won't interfere or cause load